staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -133,6 +133,20 @@ func createHandler(r rest.NamedCreater, scope RequestScope, admit admission.Inte
 			}
 		}
 
+		if scope.FieldManager != nil {
+			liveObj, err := scope.Creater.New(scope.Kind)
+			if err != nil {
+				scope.err(fmt.Errorf("failed to create new object: %v", err), w, req)
+				return
+			}
+
+			obj, err = scope.FieldManager.Update(liveObj, obj, "create")
+			if err != nil {
+				scope.err(fmt.Errorf("failed to update object managed fields: %v", err), w, req)
```

Comment:
Until we've exhaustively tested using every field from every schema, I don't think we can make this error fail a request. Alpha features mustn't break existing behavior.